src/DynamoCoreWpf/ViewModels/PackageManager/PackageViewModel.cs

```diff
@@ -30,6 +31,80 @@ public bool HasAdditionalAssemblies
             get { return Model.LoadedAssemblies.Any(x => !x.IsNodeLibrary); }
         }
 
+        [Obsolete("This is not final. Please do not use it")]
+        public string PackageLoadStateText
+        {
+            get
+            {
+                switch (Model.LoadState.ScheduledState)
+                {
+                    case PackageLoadState.ScheduledTypes.ScheduledForLoad: return Resources.PackageStateScheduledForLoad;
+                    case PackageLoadState.ScheduledTypes.ScheduledForUnload: return Resources.PackageStateScheduledForUnload;
+                    case PackageLoadState.ScheduledTypes.ScheduledForDeletion: return Resources.PackageStateScheduledForDeletion;
+                    default:
+                        break;
+                }
+
+                switch (Model.LoadState.State)
+                {
+                    case PackageLoadState.StateTypes.Unloaded: return Resources.PackageStateUnloaded;
+                    case PackageLoadState.StateTypes.Loaded: return Resources.PackageStateLoaded;
+                    case PackageLoadState.StateTypes.Error: return Resources.PackageStateError;
+                    default:
+                        return "Unkonwn package state";
+                }
+            }
+        }
+
+        [Obsolete("This is not final. Please do not use it")]
+        public string PackageLoadStateTooltip
+        {
+            get
+            {
+                switch (Model.LoadState.ScheduledState)
+                {
+                    case PackageLoadState.ScheduledTypes.ScheduledForLoad: return Resources.PackageStateScheduledForLoadTooltip;
+                    case PackageLoadState.ScheduledTypes.ScheduledForUnload: return Resources.PackageStateScheduledForUnloadTooltip;
+                    case PackageLoadState.ScheduledTypes.ScheduledForDeletion: return Resources.PackageStateScheduledForDeletionTooltip;
+                    default:
+                        break;
+                }
+
+                switch (Model.LoadState.State)
+                {
+                    case PackageLoadState.StateTypes.Unloaded: return Resources.PackageStateUnloadedTooltip;
+                    case PackageLoadState.StateTypes.Loaded: return Resources.PackageStateLoadedTooltip;
+                    case PackageLoadState.StateTypes.Error: return string.Format(Resources.PackageStateErrorTooltip, Model.LoadState.ErrorMessage);
+                    default:
+                        return "Unkonwn package state";
```

Comment:
typo: unknown
Maybe this needs to be a resource string too.

Reply:
This is not supposed to be a valid state. It is only useful for debugging ... 
Do you think it is useful to be localized ?